gpudirect-tcpxo/topology-scheduler/schedule-daemon.py

```diff
@@ -137,14 +135,8 @@ def find_schedulable_nodes(nodes, pods, tolerated_taints):
     node_name = node.metadata.name
     node_labels = node.metadata.labels
 
-    if 'cloud.google.com/gke-placement-group' not in node_labels:
-      print(
-          f'Skipping node {node_name} because it does not have topology'
-          ' metadata'
-      )
-      continue
```

Comment:
Can you break out the removal of this to another PR? I'm not against cleaning this up, but it was a hard requirement and is different than the readiness check as far as changes go.